.tekton/retag-pipeline.yaml

```diff
@@ -154,12 +154,29 @@ spec:
         - name: name
           value: determine-dependency-image-tag
         - name: bundle
-          value: quay.io/rhacs-eng/konflux-tasks:latest@sha256:d5c49395c67b924da06f5bc3dab1efd224184e86a1f7006b7ebb1cc342032dd5
+          value: quay.io/rhacs-eng/konflux-tasks:latest@sha256:8ebb5d5589a23798f7c3f049d9ba42f726c5a39314f1cb10b10f9466e89d0dca
         - name: kind
           value: task
       resolver: bundles
 
+  - name: wait-for-image
```

Comment:
Please check timeouts for the retag pipelines. They were always quick because all tasks execute quickly when the images are there.

It makes no sense to me to wait for images to appear in these retag pipelines, it's better to fail them early. Therefore, I'm suggesting to set a small timeout to this `wait-for-image` task in each of retag pipelines so that it's just enough to check for the presence.

It would then be helpful if you can add a comment explaining that we have this task to verify not only that the images are there but also that they successfully passed their build pipelines. In this case, the comment on `retag-image`'s `runAfter` could perhaps be redundant.

Reply:
https://github.com/stackrox/stackrox/pull/15552/commits/5e44d040f3513e81f6ef8250f339d03738155f9d